### ISSUE TITLE
[FR] Teletype support

### ISSUE THREAD

**Issue by codic12** [bus-stop/x-terminal#55]
The only thing that is preventing me from using atom instead of vscode full time, at the moment, is the lack of sharing a terminal with another person. VSCode can do this with the live share extension.
Why not integrate with teletype?
I currently use https://tmate.io/ which works amazingly but teletype support would be a step further to make it feel native

**Comment by codic12** [comment]
Haha, so I assume that this would be possible with the constraints of teletype?

**Comment by UziTech** [comment]
Not sure. Teletype doesn't have very good documentation. (Or any at all)

**Comment by the-j0k3r** [comment]
Atom describes how to add support for teletype in no ambiguous manner.

https://teletype.atom.io/#getting-started

Or https://github.com/atom/teletype

Or Open atom search for package teletype and install.

Or open a terminal and type `apm install teletype`

Its not up to any terminal to include this anyway as its a editor side modification.

**Comment by UziTech** [comment]
Teletype also provides a service for packages to sync information. That is what this issue is about. Syncing a terminal like vscode does.

The problem is there is no documentation for this service.

I think we would need to replace pty with a teletype service to get it working.

**Comment by codic12** [comment]
Yes, this issue is about syncing information using the Teletype API. That sucks, should I close this issue in that case?

**Comment by UziTech** [comment]
No, we should keep this open. This is something I would like to figure out eventually.

**Comment by the-j0k3r** [comment]
And please dont forget that people who dont want any of this shouldn't be forced to load this service whatever it will end up being.

e.g people who dont want any remote sharing services when they dont have any for security reasons.
